packages/discord.js/src/managers/GuildChannelManager.js

```diff
@@ -180,14 +181,15 @@ class GuildChannelManager extends CachedManager {
    * @returns {Promise<Webhook>} Returns the created Webhook
    * @example
    * // Create a webhook for the current channel
-   * guild.channels.createWebhook('222197033908436994', 'Snek', {
+   * guild.channels.createWebhook('222197033908436994', {
+   *   name: 'Hello!',
    *   avatar: 'https://i.imgur.com/mI8XcpG.jpg',
    *   reason: 'Needed a cool new Webhook'
    * })
    *   .then(console.log)
    *   .catch(console.error)
    */
-  async createWebhook(channel, name, { avatar, reason } = {}) {
+  async createWebhook(channel, { name, avatar, reason } = {}) {
```

Comment:
```suggestion
  async createWebhook({ channel, name, avatar, reason } = {}) {
```

Reply:
I agree. Btw in your change channel is required but adding `= {}` the whole object will become optional. No?

Reply:
The channel name is not optional either, so moving the name to the options object it would already make it required.

So the `= {}` should be removed